src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNuSpec.cs

```diff
@@ -197,7 +197,7 @@ private List<ManifestFile> GetManifestFiles()
                     select new ManifestFile()
                     {
                         Source = f.GetMetadata(Metadata.FileSource),
-                        Target = f.GetMetadata(Metadata.FileTarget),
+                        Target = f.GetMetadata(Metadata.FileTarget).Replace('/', Path.DirectorySeparatorChar),
```

Comment:
I'm not certain I understand your question.  We generate the target path with '/'.  We could do '\\', but that's just flipping the problem.  The issue is that we need to generate a nuspec file with the proper path separator for the platform we're running on or else the PathResolver will fail to match assets.